synctl

```diff
@@ -861,6 +861,60 @@ class SyntheticPoP(Base):
                 f"Failed to get locations, status code {retrieve_res.status_code}")
             sys.exit(-1)
 
+    def get_location_summary_list(self):
+        """ --request POST 'http://{host}/api/synthetics/results/locationsummarylist'
+        --header "Authorization: apiToken <YourToken>" -i
+        --header "Content-Type: application/json"
+        -d '{
+             "pagincurlation": {
+             "page": 1,
+             "pageSize": 200
+            },
+             "timeFrame": {
+             "to": 0,
+             "windowSize": 3600000
+            }
+         }'
+         """
+
+        host = self.auth["host"]
+        token = self.auth["token"]
+        self.check_host_and_token(host, token)
+
+        headers = {
+            'Content-Type': 'application/json',
+            "Authorization": f"apiToken {token}"
+        }
+
+        summary_config = {
+            "pagination": {
+                "page": 1,
+                "pageSize": 200
+            },
+            "timeFrame": {
+                "to": 0,
+                "windowSize": 3600000
+            }
+        }
+        request_url = f"{host}/api/synthetics/results/locationsummarylist"
+        retrieve_res = requests.post(request_url,
+                                     headers=headers,
+                                     data=json.dumps(summary_config),
+                                     timeout=60,
+                                     verify=self.insecure)
+
+        if retrieve_res.status_code == 200:
```

Comment:
/api/synthetics/results/locationsummarylist support multiple pages if totalHits are more than 200(max page size). See API docs https://instana.github.io/openapi/#operation/getLocationSummaryList for more details.

Reply:
In this url `http://{host}/api/synthetics/results/locationsummarylist`, http is hardcoded, it can be http or https, actually https is used mostly

Reply:
It doesn't matter, line 865 is a comment not real http URL, request url is in line 899 `request_url = f"{host}/api/synthetics/results/locationsummarylist"`, `host` depends on users' config.